google/cloud/bigtable/bytes.h

```diff
@@ -44,6 +44,9 @@ class Bytes {
   /// An empty sequence.
   Bytes() = default;
 
+  /// Strips the null-terminator character from input bytes.
+  explicit Bytes(char const* bytes) : bytes_(bytes) {}
```

Comment:
Is it possible to have a unit test to verify the fix this change is aiming to make?

Reply:
The updates I had to make to the existing test is probably the only way to verify the new behavior.